### ISSUE TITLE
add keyboard shortcuts for accessibility

### ISSUE THREAD

**Issue by neonpixii** [klembot/twinejs#1089]
### Is your feature request related to a problem? Please describe.

the software lacks basic accessibility requirements for a large proportion of users.  I have limited hand usage and reduced
dexterity due to injuries, and primarily interact with my computer through voice control software (talon).  mouse driven user interfaces are difficult for me to use and often painful, aggravating my injuries, whereas I can make voice bindings for keyboard shortcuts quite easily.

there are a very large number of computer users who would benefit from this software being usable solely with the keyboard.  they include people like me who are navigating or trying to heal from injuries, including repetitive strain injuries (which are quite common in the computer world), people with disabilities, people with reduced or limited dexterity or mobility, as well as people who simply prefer keyboard input or find it more convenient

### Describe the solution you'd like.

comprehensive keyboard shortcuts that enable full functionality of the program without requiring mouse usage. bonus points if there rebindable

### Describe alternatives you've considered.

there is no substitute for accessibility. the closest I've seen is twee, but its install process is broken and I'd prefer the graph based view of twine anyways

### Additional context on this suggestion.

I heard a lot about how accessible twine was, but was really disappointed to find myself excluded from using this software because the accessibility needs of me and a large swath of other users were not considered

### Presubmission checklist

- [ ] I am interested in working on code that would implement this feature request. (This is not required to submit a suggestion.)
- [X] I have done a search and believe that an issue does not already exist for this idea in the GitHub repository.
- [X] I have read and agree to abide by this project's Code of Conduct.

**Comment by klembot** [comment]
2.4 has some improvements with regards to keyboard navigability but it isn't completely there yet. What I'd like to do is update this issue when I think things are in a better place to get your feedback on it, and ensure that it actually is usable.

Being able to rebind keyboard shortcuts was raised in #1082, see my comments there.

**Comment by neonpixii** [comment]
that means a lot to me! I'll keep an eye on this
